python/main.py

```diff
@@ -0,0 +1,28 @@
+from fastapi import FastAPI, UploadFile, File, Request
+from fastapi.responses import HTMLResponse, RedirectResponse
+from fastapi.templating import Jinja2Templates
+import uvicorn
+from upload_endpoint import upload_image_to_s3
+from starlette import status
+
+
+
+app = FastAPI()
+templates = Jinja2Templates(directory="htmldirectory")
+
+
+
+## file upload
+@app.post("/submit")
+async def root(receipt_image: UploadFile = File(...)):
```

Comment:
i would rename the function. 
and maybe use another path like "/upload" or "/image"